src/shared/modals/custom-filter/CollectionsFilter.js

```diff
@@ -14,15 +14,15 @@ const CollectionsFilter = (props) => {
 
   return (
     <React.Fragment>
-      <button onClick={handleClick}>
+      <button className="modal__col-btn" onClick={handleClick}>
```

Comment:
I would make the class for this `modal__collections-btn` as `col` is usually short for columns.